Path-Check/gaen-mobile
# Upload Keys to Verification Server on Positive Diagnosis

#### Description:
Allows users to report a positive test result and upload their keys to the verification server.

#### How to test:
1. Install app and ensure it has been installed for at least 30 minutes. Enough time to generate exposure keys.
2. Start Report Positive Test Result flow.
3. Check logcat and observe keys have been uploaded successfully.
